components/CostsCard.vue

```diff
@@ -61,6 +60,9 @@
           @mouseover="hideTooltips = false"
         />
       </div>
+      <p class="fw-lighter vsl-display">
+        * Value of statistical life: ${{ formatCurrency(appStore.currentScenario.result.data?.average_vsl || 0) }} Int'l$
+      </p>
```

Comment:
I think this is probably one of those situations where you ought to be able to assume that if we're showing this element then the result data exists. We shouldn't ever be showing 0 here!